src/aap_eda/api/serializers/__init__.py

```diff
@@ -126,9 +121,6 @@
     "CredentialCreateSerializer",
     "CredentialRefSerializer",
     # credential type
-    "CredentialTypeSerializer",
```

Comment:
@bzwei I don't think this can be removed it was added in this PR https://issues.redhat.com/browse/AAP-22997

Reply:
Yes, we need these serializers